File: lvl 3.py
# 
# rows = 5
# while rows >= 1:
#     print("*" * rows)
#     rows -= 1

# 

# def luasPersegi(sisi):
#     return sisi * sisi

# print(luasPersegi(4))

# def cek_ganjil_genap(angka):
#     if angka % 2 == 0:
#         print("genap")
#     else:
#         print("ganjil")

# cek_ganjil_genap(6)
# cek_ganjil_genap(7)

def konversiSuhu(celcius):
    fahrenheit = celcius * 1.8 + 32
    return fahrenheit

hasil = konversiSuhu(30)
print(hasil)

File: lvl 3.js
// let nilai = "B";

// switch (nilai) {
//     case "A":
//         console.log("sangat Baik");
//         break;
//     case "B":
//         console.log("bagus");
//         break;
// default:
//     console.log("tidak diketahui");
// }

// 

// let rows = 5;
// let i = 1;
// do {
//     console.log(" ".repeat(rows-1)+"*".repeat(i))

// i++;
// }while(i <= rows);

// function luasPersegi(sisi) {
//     return sisi * sisi;
// }

// console.log(luasPersegi(4));

// function cekGanjilGenap(angka) {
//     if (angka % 2 === 0) {
//         console.log("genap");
//     } else {
//         console.log("ganjil");
//     }
// }

// cekGanjilGenap(6);
// cekGanjilGenap(7);

function konversiSuhu(celcius) {
    let fahrenheit = celcius * 1.8 + 32;
    return fahrenheit;
}

let hasil = konversiSuhu(30);
console.log(hasil);